nautobot/extras/api/serializers.py

```diff
@@ -170,8 +170,6 @@ class NautobotModelSerializer(
 class StatusModelSerializerMixin(BaseModelSerializer):
     """Mixin to add `status` choice field to model serializers."""
 
-    status = StatusSerializerField(required=True)
```

Comment:
I need some time to figure out what we are doing with `TaggedModelSerializerMixin` in this file.